website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -198,19 +210,10 @@ const useExternalApiFetcher = () => {
 
 Available options are:
 
-```ts
-type TForwardToAudiencePolicy =
-  | 'forward-url-full-path'
-  | 'forward-url-origin';
-
-type TApolloContextProxyForwardTo = {
-  uri: string;
-  headers?: Record<string, string>;
-  audiencePolicy?: TForwardToAudiencePolicy;
-};
-```
-
-For more information see [configuring the audience policy](/concepts/integrate-with-your-own-api#configuring-the-audience-policy).
+* `uri` (required): The URL to request to from the external API.
```

Comment:
nitpick: If we are going to adopt this notation, I find it useful to use `bold` for highlight required parameter and normal text for optional ones.

```
* `uri` (**required**): The URL to request to from the external API.
```